src/content/chainlink-automation/guides/migrate-to-v2.mdx

```diff
@@ -47,14 +45,33 @@ Upkeeps that are successfully migrated will show the following [transaction logs
 
 <ClickToZoom src="/images/automation/v2-migration/migration-txn.png" />
 
+### Migrating multiple upkeeps
+
+If you have multiple upkeeps to migrate, you can do so in the Chainlink Automation App.
+
+1. Click the link displays on your upkeeps dashboard. This link only displays if you have one or more upkeeps to migrate:
+
+   <ClickToZoom src="/images/automation/v2-migration/1-bulk-migration-ui.png" />
+
+1. Follow the prompts to approve and confirm the transactions to migrate your upkeep(s).
+
+   <ClickToZoom src="/images/automation/v2-migration/approve-migration.png" />
+
+### Updating permissions
```

Comment:
Is this required for bulk migration? If yes, add a step to point users to this section instructing them to update their permissions. Also, does this apply to the "Migrating using the Chainlink Automation App" section?

Reply:
It potentially applies to any type of migration, so I restructured it to reflect that. Thanks!